src/block.js

```diff
@@ -0,0 +1,9 @@
+import { defaultConfig } from './configs';
```

Comment:
**Suggestion:**

I'd like to see this file as `Block.js` instead of 'block.js' to represent the React component within.